thoth/storages/graph/postgres.py

```diff
@@ -64,6 +64,7 @@
 from thoth.common import OpenShift
 from thoth.common import map_os_name
 from thoth.common.enums import ThothAdviserIntegrationEnum
+from thoth.license_solver import detect_license
```

Comment:
This is not needed here , right
as this is already used in solver, and we would directly insert the result of solver via sync.